Business bulk upload

pswk1 opened the pull request:
- First step of onboarding flow has owner's business info pre-populated
- Owner can make changes/corrections if necessary, and continuing to the next step will update to reflect those changes
- Going back to the bizinfo step will reflect those changes as well
- Each step of the template upload flow has a link to the "Why am I uploading this?" view

pswk1 committed:
feat(business onboarding): bizinfo form

- first step of onboarding flow has a pre-populated form of the owner's business info
- owner can make changes and/or corrections and continuing to the next step will update those changes

pswk1 committed:
fix(business details): bizinfo cache update

- In onboarding flow, if user goes back a page, bizinfo is updated

pswk1 committed:
feat(business details): user use case link

- At each template import step in the onboarding flow, the "Why am I entering this data?" link will open a new tab outlining the use cases for each upload

pswk1 committed:
refactor(business details): minor refactoring

- remove unused classes and minor style fix

pswk1 committed:
Apply formatting changes

pswk1 committed:
Merge branch 'bulk-upload-modals' into business-bulk-upload

pswk1 committed:
Apply formatting changes

pswk1 committed:
fix(biz hours form): fix from merge conflict

- update BusinessHoursForm to current code from bulk-upload-modals branch

pswk1 committed:
Merge branch 'business-bulk-upload' of github.com:omou-org/front-end into business-bulk-upload

pswk1 committed:
feat(biz hours form): update hours on onboarding

- business hours update in onboarding flow

dhuangg commented on src/components/FeatureViews/Onboarding/BusinessInfo.js:
```diff
@@ -22,48 +23,93 @@ const useStyles = makeStyles(() => ({
+    const [updateData] = useMutation(UPDATE_BUSINESS, {
+        // onCompleted: (data) => {
+        //     console.log(data);
+        // },
```
delete comments

dhuangg commented on src/components/FeatureViews/Onboarding/ImportFlow.js:
```diff
@@ -43,9 +43,9 @@ const ImportFlow = () => {
     useEffect(() => {
-        const currentStep = Number(urlQuery.get('step')) - 1;
+        // const currentStep = Number(urlQuery.get('step')) - 1;
+        const currentStep = Number(urlQuery.get('step'));
```
we need to do step-1 to convert it to the step indexing so change this back to line 47

dhuangg reviewed (changes requested):
small clean-up change + revert back to an old line of code~

pswk1 committed:
fix(business onboarding): stepper error

- welcome step of onboarding correctly links to first step in business onboarding flow
- steps start at 1 instead of 0 for more logical UX

dhuangg reviewed (approved):
lezzgooooo

pswk1 committed:
Merge branch 'bulk-upload-modals' into business-bulk-upload

pswk1 committed:
Apply formatting changes
